mountain-climbing/src/main/java/com/example/mountainclimbing/MountainClimbingApplication.java

```diff
@@ -12,4 +16,23 @@
 	public static void main(String[] args) {
 		SpringApplication.run(MountainClimbingApplication.class, args);
 	}
+	
+	public class MailSender {
```

Comment:
Rename MaliSender to MailSenderConfiguration and set as static class. Annotate class with @Configuration but then you need to delete spring.mailx. * prefix from application.properties of vice verca